pkg/plugins/resources/gittag/main.go

```diff
@@ -18,6 +18,10 @@ type Spec struct {
 	VersionFilter version.Filter `yaml:",omitempty"`
 	// Message associated to the git tag
 	Message string `yaml:",omitempty"`
+	// Key of the tag object to retrieve, default is tag "name"
+	//  filters are always against tag name, this only controls the output
+	//  current values are 'name' and 'hash'
```

Comment:
Technical limitation of the library that I am relying on for generating the jsonschema (https://github.com/invopop/jsonschema#using-go-comments). The comment must be on one line.

Otherwise we'll just see "current values are 'name' and ''hash" both from IDE auto completation and documentation website

That being said, I don't see that constrain anymore so maybe I need to give it another try

Reply:
I went ahead and moved this into a single line.